pytorch_lightning/callbacks/gpu_stats_monitor.py

```diff
@@ -0,0 +1,164 @@
+"""
+GPU Stats Monitor
+====================
+
+Monitor and logs GPU stats during training.
+
+"""
+
+import os
+import shutil
+import subprocess
+import time
+from argparse import Namespace
+
+from pytorch_lightning.callbacks.base import Callback
+from pytorch_lightning.utilities import rank_zero_warn
+from pytorch_lightning.utilities.exceptions import MisconfigurationException
+
+
+class GPUStatsMonitor(Callback):
```

Comment:
how it would work on multi-node if the GPU indexes would be the same so the metrics would overlap?